core/src/main/java/com/flipkart/varadhi/core/cluster/messages/SubscriptionOperation.java

```diff
@@ -0,0 +1,45 @@
+package com.flipkart.varadhi.core.cluster.messages;
+
+
+import com.flipkart.varadhi.entities.VaradhiSubscription;
+import lombok.Data;
+
+import java.util.UUID;
+
+@Data
+public class SubscriptionOperation {
+    public static enum Kind {
+        CREATE, START, STOP, DELETE, UPDATE
```

Comment:
why are there crud operations here?

Reply:
removing for now, will add later if needed to persist create/delete.